org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/editor/MavenTargetLocationWizard.java

```diff
@@ -121,13 +121,10 @@ public void createControl(Composite parent) {
 					metadata.setSelection(new StructuredSelection(MavenTargetLocation.DEFAULT_METADATA_MODE));
 					bndInstructions = new BNDInstructions("", null); //$NON-NLS-1$
 					includeSource.setSelection(true);
-					String clipboardError = clipboardParser.getError();
+					Exception clipboardError = clipboardParser.getError();
 					if (clipboardError != null) {
-						setMessage(MessageFormat.format(Messages.MavenTargetLocationWizard_11, clipboardError),
-								WARNING);
-						parent.getDisplay().timerExec((int) TimeUnit.SECONDS.toMillis(10), () -> {
-							setMessage(null);
-						});
+						Platform.getLog(MavenTargetLocationWizard.class).warn(MessageFormat
```

Comment:
In most cases, the clipboard wouldn't contain a XML document; which means in most cases, this warning will be shown, even for regular usage. I don't think this should be logged at all, a clipboard that doesn't contain GAV is a normal thing.

Reply:
I do a poor-man-precheck by requiring that the clipboard:
a) must contain text
b) the text must start with an '<' char
that should guard against most common non-xml clipboard content. E.g. my error log is filled with many "UI freeze", "duplicate keybinding" stuff so I would be happy to at least see somewhere something useful if I add a new target entry and it does not happen as expected :-)

Reply:
OK, I missed the preliminary check. I think it's OK to log in this restricted cases.